Gems/ROS2/Code/Source/Lidar/LidarTemplateUtils.cpp

```diff
@@ -18,21 +18,128 @@ namespace ROS2
+                        /*.m_distanceNoiseStdDevBase = */ 0.002f,
+                        /*.m_distanceNoiseStdDevRisePerMeter = */ 0.0008f,
+                    },
+                },
+            },
+            {
+                LidarTemplate::LidarModel::Ouster_OS2_64,
+                {
+                    /*.m_model = */ LidarTemplate::LidarModel::Ouster_OS1_64,
+                    /*.m_name = */ "Ouster OS1-64",
+                    /*.m_minHAngle = */ -180.0f,
+                    /*.m_maxHAngle = */ 180.0f,
+                    /*.m_minVAngle = */ 11.25f,
+                    /*.m_maxVAngle = */ -11.25f,
+                    /*.m_layers = */ 64,
+                    /*.m_numberOfIncrements = */ 2048,
+                    /*.m_maxRange = */ 225.0f,
+                    /*.m_noiseParameters = */
+                    {
+                        /*.m_angularNoiseStdDev = */ 0.0f,
+                        /*.m_distanceNoiseStdDevBase = */ 0.006f,
+                        /*.m_distanceNoiseStdDevRisePerMeter = */ 0.001f,
+                    },
+                },
+            },
+            {
+                LidarTemplate::LidarModel::Velodyne_Puck,
+                {
+                    /*.m_model = */ LidarTemplate::LidarModel::Velodyne_Puck,
+                    /*.m_name = */ "Velodyne Puck (VLP-16)",
+                    /*.m_minHAngle = */ -180.0f,
+                    /*.m_maxHAngle = */ 180.0f,
+                    /*.m_minVAngle = */ 15.0f,
+                    /*.m_maxVAngle = */ -15.0f,
+                    /*.m_layers = */ 16,
+                    /*.m_numberOfIncrements = */ 1800, // For 0.2 angular resolution
+                    /*.m_maxRange = */ 100.0f,
+                    /*.m_noiseParameters = */
+                    {
+                        /*.m_angularNoiseStdDev = */ 0.0f,
+                        /*.m_distanceNoiseStdDevBase = */ 0.03f,
+                        /*.m_distanceNoiseStdDevRisePerMeter = */ 0.001f,
+                    },
+                },
+            },
+            {
+                LidarTemplate::LidarModel::Velodyne_HDL_32E,
+                {
+                    /*.m_model = */ LidarTemplate::LidarModel::Velodyne_HDL_32E,
+                    /*.m_name = */ "Velodyne HDL-32E",
+                    /*.m_minHAngle = */ -180.0f,
+                    /*.m_maxHAngle = */ 180.0f,
+                    /*.m_minVAngle = */ 10.67f,
+                    /*.m_maxVAngle = */ -30.67f,
+                    /*.m_layers = */ 32,
+                    /*.m_numberOfIncrements = */ 1800, // For 0.2 angular resolution
+                    /*.m_maxRange = */ 100.0f,
+                    /*.m_noiseParameters = */
+                    {
+                        /*.m_angularNoiseStdDev = */ 0.0f,
+                        /*.m_distanceNoiseStdDevBase = */ 0.02f,
+                        /*.m_distanceNoiseStdDevRisePerMeter = */ 0.001f,
+                    },
```

Comment:
+1 to this style I like it! As a stand-in for designated initializers it's not bad! 🙂